src/GafferSceneUI/Inspector.cpp

```diff
@@ -289,7 +289,21 @@ void Inspector::inspectHistoryWalk( const GafferScene::SceneAlgo::History *histo
 
 					// See if we can use it for editing
 
-					if( !result->m_editScope || node->ancestor<EditScope>() == result->m_editScope )
+					if( !result->m_editScope && node->ancestor<EditScope>() )
+					{
+						// We don't allow editing if the user hasn't requested a specific scope
+						// (they have selected "None" from the Menu) and the upstream edit is
+						// inside _any_ EditScope.
+						result->m_editFunction = fmt::format(
+							"Edit is in an edit scope. Change scope to {} to edit.",
```

Comment:
Suggestion :

> *Source* is in an *EditScope*. Change scope to {} to edit.

I prefer Source because to me Edit implies a tweak to an already-existing value, but it's possible for the source to be the originator of the value (even when in an EditScope). It also reduces the repetition of edit, which I think helps readability a bit.

And I think in this case we're talking about an EditScope - the concrete node type. My thinking is that "edit scope" is for cases where we're referring to the current scope - either an EditScope or None.